examples/scripts/dpo_online.py

```diff
@@ -57,6 +59,8 @@
 from trl.trainer.utils import SIMPLE_CHAT_TEMPLATE
 
 
+JUDGES = {"pair_rm": PairRMJudge, "openai": OpenAIPairwiseJudge}
```

Comment:
Should we include HF ones too?
```suggestion
JUDGES = {"pair_rm": PairRMJudge, "openai": OpenAIPairwiseJudge, "hf": HfPairwiseJudge}
```

Reply:
LGTM. Of course, if a user needs to use these judges with parameters other than the default, they will have to modify the script. This is the case, for example, if they want to use a model other than Meta-Llama-3-70B-Instruct for `HfPairwiseJudge`.

Reply:
fbec78a